MORE-INFO.md

```diff
@@ -0,0 +1,86 @@
+# Additional Info - Universal Connect Widget
+
+## Table of Contents
+
+- [Preferences and Aggregator Credentials](#preferences-and-aggregator-credentials)
+- [Sensitive information](#sensitive-information)
+- [Deploying a Production Instance](#deploying-a-production-instance)
+- [Authentication](#authentication)
+- [API Documentation](#api-documentation)
+- [Cached Data](#cached-data)
+- [Troubleshooting](#troubleshooting)
+- [Redis](#redis)
+- [Architecture Decision Records](#architecture-decision-records)
+- [Monorepo](#monorepo)
+
+## Preferences and Aggregator Credentials
```

Comment:
I think this section should live in the main README. npm run dev won't work without it. We should call out that they need to sign up with each provider and get the credentials